schema.json

```diff
@@ -0,0 +1,226 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://alpheios.net/schemas/apis/user-word/item.json",
+  "type": "object",
+  "title": "Schema for a User Word Item",
+  "definitions": {
+    "targetWord": {
+      "$id": "#/definitions/targetWord",
+      "type": "string",
+      "title": "Target Word Schema",
+      "examples": [
+        "cespitum"
+      ],
+      "pattern": "^(.+)$"
+    },
+    "languageCode": {
+      "$id": "#/definitions/languageCode",
+      "type": "string",
+      "title": "Language Code Schema",
+      "examples": [
+        "lat"
+      ],
+      "pattern": "^\\w\\w\\w$"
+    },
+    "createdDT": {
+      "$id": "#/definitions/createdDT",
+      "type": "string",
+      "title": "Created DT Schema",
+      "examples": [
+        "2019/10/03 @ 12:56:58"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "selector": {
+      "$id": "#/definitions/imported/oa/selector",
+      "type": "object",
+      "title": "Selector Schema (see http://www.w3.org/ns/oa#TextQuoteSelector )",
+      "required": [
+        "type",
+        "exact",
+        "prefix",
+        "suffix",
+        "languageCode"
+      ],
+      "properties": {
+        "type": {
+          "$id": "#/definitions/imported/oa/type",
+          "type": "string",
+          "title": "Type Schema",
+          "default": "TextQuoteSelector",
+          "examples": [
+            "TextQuoteSelector"
+          ],
+          "pattern": "^TextQuoteSelector$"
+        },
+        "exact": {
+          "$id": "#/definitions/imported/oa/exact",
+          "type": "string",
+          "title": "Exact Schema",
+          "examples": [
+            "cespitum"
+          ],
+          "pattern": "^(.+)$"
+        },
+        "prefix": {
+          "$id": "#/definitions/imported/oa/prefix",
+          "type": "string",
+          "title": "Prefix Schema",
+          "default": "",
+          "examples": [
+            "words that come before"
+          ],
+          "pattern": "^(.*)$"
+        },
+        "suffix": {
+          "$id": "#/definitions/imported/oa/suffix",
+          "type": "string",
+          "title": "Suffix Schema",
+          "default": "",
+          "examples": [
+            "words that come after"
+          ],
+          "pattern": "^(.*)$"
+        },
+        "languageCode": {
+          "$ref": "#/definitions/languageCode"
+        }
+      }
+    }
+  },
+  "required": [
+    "ID",
+    "listID",
+    "userID",
+    "languageCode",
+    "targetWord",
+    "important",
+    "createdDT",
+    "context"
+  ],
+  "optional": [
+    "homonym"
+  ],
+  "properties": {
+    "ID": {
+      "$id": "#/properties/ID",
+      "type": "string",
+      "title": "Word Item ID Schema",
+      "examples": [
+        "lat-cespitum"
+      ],
+      "pattern": "^\\w\\w\\w-(.+)$"
```

Comment:
I'm wondering if the first part should match lower-case alphanumeric characters only

Reply:
hmm. possibly.